light-client/tests/integration.rs

```diff
@@ -0,0 +1,125 @@
+//! Light Client integration tests.
+//!
+//! These are all ignored by default, since they test against running
+//! `tendermint node --proxy_app=kvstore`. They can be run using:
+//!
+//! ```
+//! cargo test -- --ignored
+//! ```
+
+use tendermint_light_client::{
+    components::{
+        clock::SystemClock,
+        io::{AtHeight, Io, IoError, ProdIo},
+        scheduler,
+        verifier::ProdVerifier,
+    },
+    evidence::{Evidence, EvidenceReporter},
+    fork_detector::ProdForkDetector,
+    light_client::{self, LightClient},
+    peer_list::PeerList,
+    state::State,
+    store::{memory::MemoryStore, LightStore},
+    supervisor::{Handle, Instance, Supervisor},
+    types::{PeerId, Status, TrustThreshold},
+};
```

Comment:
```suggestion
use tendermint_light_client::components::clock::SystemClock;
use tendermint_light_client::components::io::{AtHeight, Io, IoError, ProdIo};
use tendermint_light_client::components::scheduler;
use tendermint_light_client::components::verifier::ProdVerifier;
use tendermint_light_client::evidence::{Evidence, EvidenceReporter};
use tendermint_light_client::fork_detector::ProdForkDetector;
use tendermint_light_client::light_client::{self, LightClient};
use tendermint_light_client::peer_list::PeerList;
use tendermint_light_client::state::State;
use tendermint_light_client::store::{memory::MemoryStore, LightStore};
use tendermint_light_client::supervisor::{Handle, Instance, Supervisor};
use tendermint_light_client::types::{PeerId, Status, TrustThreshold};
```

This style of imports is way easier to edit.

Reply:
Yeah we are not very consistent with the style of imports we use. I started switching to the single import style since that's apparently what the Rust community in general is moving towards, but I don't feel strongly either way. Let's track this in #433.

Reply:
Since this import style shows up in most places in the `light-client` crate already, I am going to leave it like this until we settle on an import style in #433.

Reply:
A follow-up on this sounds perfect. Thanks for capturing it.